drivers/gles3/storage/material_storage.cpp

```diff
@@ -3452,6 +3452,10 @@ void SceneShaderData::set_code(const String &p_code) {
 		blend_mode = BLEND_MODE_ALPHA_TO_COVERAGE;
 	}
 
+	if (blend_mode == BLEND_MODE_ADD || blend_mode == BLEND_MODE_SUB || blend_mode == BLEND_MODE_MUL) {
+		uses_blend_alpha = true; //force alpha used because of blend
```

Comment:
```suggestion
		uses_blend_alpha = true; // Force alpha used because of blend.
```
Our style guide requires comments to start with a capital and end with a period. I know this comment was a copy paste from a line already in the engine, but we do try to enforce the style guide on all new comments

Reply:
Thanks for pointing this out! Makes perfect sense :)